### PR TITLE
Add build & publish workflows

### LINKED ISSUE #12: Publish extension to npm

### PR DESCRIPTION
**Pull request by sgraband**
Added 2 workflows for building and publishing.
Ignored 2 linting rules that conflict with the formatter.
Moved server.jar back to extension.

Resolves #12.

### CODE REVIEW

**Comment by sgraband** [comment]
Publish script was tested and successful: https://github.com/eclipsesource/model-compare/runs/5771278644. Resulted in this version: https://www.npmjs.com/package/@eclipsesource/comparison-extension/v/0.0.1-next.20220331.125641.

File: client/examples/coffee-example/coffee-comparison-extension/src/browser/coffee-comparison-frontend-module.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
-import { ComparisonExtensionConfiguration } from 'comparison-extension/lib/browser/comparison-extension-configuration';
+import { ComparisonExtensionConfiguration } from '@eclipsesource/comparison-extension/lib/browser/comparison-extension-configuration';
```
Comment (eneufeld) [inline client/examples/coffee-example/coffee-comparison-extension/src/browser/coffee-comparison-frontend-module.ts:16]: is the export wrong? why the weird import?

**Review by eneufeld — APPROVED** [review]
Looks fine, only the weird imports I marked inline could be improved in the future, please open an issue if there is none yet

File: client/examples/coffee-example/coffee-comparison-extension/src/browser/coffee-comparison-frontend-module.ts (hunk 1 above)
Comment (sgraband) [reply client/examples/coffee-example/coffee-comparison-extension/src/browser/coffee-comparison-frontend-module.ts:16]: I created a follow up issue for this: #21.

**Pull request merged**